tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
```

Comment:
I chose 1,000 based on the performance of selecting 1,000 source records with related items, but I'll change it to 10,000 if you think it's a more practical number.

As far as lifting it to a constant, I think it's a reasonable idea if it were used in more than just this view, but I don't see even another view that calls `perform_query`. My main reason for putting it here initially was to separate the query limitations enforced by the model from those enforced by the view. If the limit were to be enforced by more than one view, than it might arguably be better for the views that need the limit to extend from a common parent class that would include this constant and anything else needed to enforce consistent behavior. But for now, I'm not sure we gain very much by moving it further away from where it's used.